FIX statement values.

juanjo-nan opened the pull request:
Task #065432

juanjo-nan committed:
FIX statement values.

Task #065432

bernatnan commented on balance_abreviado.xml:
```diff
@@ -237,8 +237,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
             <field name="code">12390</field>
             <field name="name">3. Otros deudores</field>
-            <field name="current_value">balance(44,460,470,471,472,473,5531,5533,544)</field>
-            <field name="previous_value">balance(44,460,470,471,472,473,5531,5533,544)</field>
```
No desfer els 2 moviments,

bernatnan commented on balance_abreviado.xml:
```diff
@@ -432,8 +432,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
             <field name="code">21700</field>
             <field name="name">VII. Resultado del ejercicio</field>
-            <field name="current_value">balance(129,6,7)</field>
-            <field name="previous_value">balance(129,6,7)</field>
```
No desfer els 2 moviments,

bernatnan commented on balance_normal.xml:
```diff
@@ -98,8 +98,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value">balance(204)</field>
-            <field name="previous_value">balance(204)</field>
+            <field name="current_value">balance(204) + invert(2804)</field>
+            <field name="previous_value">balance(204) + invert(2804)</field>
```
Ok

bernatnan commented on balance_normal.xml:
```diff
@@ -175,8 +175,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value">balance(212,213,214,215,216,217,218,219) + invert(2810,2812,2813,2814,2815,2816,2817,2818,2819,2912,2913,2914,2915,2916,2917,2918,2919)</field>
-            <field name="previous_value">balance(212,213,214,215,216,217,218,219) + invert(2810,2812,2813,2814,2815,2816,2817,2818,2819,2912,2913,2914,2915,2916,2917,2918,2919)</field>
+            <field name="current_value">balance(212,213,214,215,216,217,218,219) + invert(2812,2813,2814,2815,2816,2817,2818,2819,2912,2913,2914,2915,2916,2917,2918,2919)</field>
+            <field name="previous_value">balance(212,213,214,215,216,217,218,219) + invert(2812,2813,2814,2815,2816,2817,2818,2819,2912,2913,2914,2915,2916,2917,2918,2919)</field>
```
Ok. Revisar si alguna bbdd de n02 té el 2810.

juanjo-nan replied:
Tenim el compte en varis clients:
- basidelta <- 2810001 - A.A. Edificios / 2810002 - A.A. Plan Gral. Pineda de Mar
- catimatge <- 2810 - Amortización acumulada de construcciones
- diloy <- 281000000000 281000000009 281000000012 281000000013 281000000014 281000000015 281000000016 281000000017 281000000018 281000000019 281000000020 281000000021 281000000022 281000000023 281000000024 281000000025 281004000000
- palsa <- 2810 - Amortización acumulada terrenos y bienes naturales
- tasello <- 281000011 - AMORT.ACUM.INMOV.MATERIAL

bernatnan commented on balance_normal.xml:
```diff
@@ -673,8 +673,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value">balance(44,5531,5533)</field>
-            <field name="previous_value">balance(44,5531,5533)</field>
+            <field name="current_value">balance(44)</field>
+            <field name="previous_value">balance(44)</field>
```
No desfer els 2 moviments,

bernatnan commented on balance_normal.xml:
```diff
@@ -712,8 +712,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value">balance(4700,4708,471,472,473)</field>
-            <field name="previous_value">balance(4700,4708,471,472,473)</field>
+            <field name="current_value">balance(4700,4708,471,472)</field>
+            <field name="previous_value">balance(4700,4708,471,472)</field>
```
No desfer els 2 moviments,

bernatnan commented on balance_normal.xml:
```diff
@@ -891,8 +891,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value">balance(5355,545,548) + debit(550,551,554,5525) + balance(565,566)</field>
-            <field name="previous_value">balance(5355,545,548) + debit(550,551,554,5525) + balance(565,566)</field>
+            <field name="current_value">balance(5355,545,548) + debit(551,554,5525) + balance(565,566)</field>
+            <field name="previous_value">balance(5355,545,548) + debit(551,554,5525) + balance(565,566)</field>
```
No desfer els 2 moviments,

bernatnan commented on balance_normal.xml:
```diff
@@ -1564,8 +1564,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value">invert(1034,1044,190,192) + balance(194,509,5115,5135,5145,521,522,523,525,526,528) + credit(550,551,554,5525) + balance(5530,5532,555,5565,5566,560,561,569)</field>
-            <field name="previous_value">invert(1034,1044,190,192) + balance(194,509,5115,5135,5145,521,522,523,525,526,528) + credit(550,551,554,5525) + balance(5530,5532,555,5565,5566,560,561,569)</field>
+            <field name="current_value">invert(1034,1044,190,192) + balance(194,509,5115,5135,5145,521,522,523,525,526,528) + credit(551,5525) + balance(555,5565,5566,560,561,569)</field>
+            <field name="previous_value">invert(1034,1044,190,192) + balance(194,509,5115,5135,5145,521,522,523,525,526,528) + credit(551,5525) + balance(555,5565,5566,560,561,569)</field>
```
No desfer els 2 moviments,

bernatnan commented on estado_ingresos_gastos_normal.xml:
```diff
@@ -24,22 +24,12 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value"></field>
-            <field name="previous_value"></field>
+            <field name="current_value">invert(802) + balance(902, 993, 994)</field>
+            <field name="previous_value">invert(802) + balance(902, 993, 994)</field>
```
Ok

bernatnan commented on pyg_abreviado.xml:
```diff
@@ -115,8 +115,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value"></field>
-            <field name="previous_value"></field>
+            <field name="current_value">balance(746)</field>
+            <field name="previous_value">balance(746)</field>
```
Revisar que no estigui dns de "a) Imputación de subvenciones, donaciones y legados de carácter financiero"

juanjo-nan replied:
Esta dins, pero no existeix al document del BOE:
https://github.com/NaN-tic/trytond-account_financial_statement_es/blob/e66abdfc38a876b28d69cd84885b9f501f61b83e/pyg_abreviado.xml#L188-L189

bernatnan commented on pyg_abreviado.xml:
```diff
@@ -175,8 +175,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value"></field>
-            <field name="previous_value"></field>
+            <field name="current_value">balance(760,761,762,767,769)</field>
+            <field name="previous_value">balance(760,761,762,767,769)</field>
```
Revisar que no estigui dns de "a) Imputación de subvenciones, donaciones y legados de carácter financiero"

juanjo-nan replied:
NO esta dins.

bernatnan commented on pyg_normal.xml:
```diff
@@ -298,8 +298,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value"></field>
-            <field name="previous_value"></field>
+            <field name="current_value">balance(746)</field>
+            <field name="previous_value">balance(746)</field>
```
Revisar que no estigui dns de "a) Imputación de subvenciones, donaciones y legados de carácter financiero"

juanjo-nan replied:
Esta dins, pero no existeix al document del BOE:
https://github.com/NaN-tic/trytond-account_financial_statement_es/blob/e66abdfc38a876b28d69cd84885b9f501f61b83e/pyg_abreviado.xml#L188-L189

bernatnan commented on pyg_pymes.xml:
```diff
@@ -87,8 +87,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value">invert(64)</field>
-            <field name="previous_value">invert(64)</field>
+            <field name="current_value">invert(64) + balance(7950,7957)</field>
+            <field name="previous_value">invert(64) + balance(7950,7957)</field>
```
Revisar que no estigui dns de "a) Imputación de subvenciones, donaciones y legados de carácter financiero"

juanjo-nan replied:
NO esta dins.

bernatnan commented on pyg_pymes.xml:
```diff
@@ -127,8 +127,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value">balance(7951,7952,7955)</field>
-            <field name="previous_value">balance(7951,7952,7955)</field>
+            <field name="current_value">balance(7951,7952,7955,7956)</field>
+            <field name="previous_value">balance(7951,7952,7955,7956)</field>
```
Revisar que no estigui dns de "a) Imputación de subvenciones, donaciones y legados de carácter financiero"

juanjo-nan replied:
No está dins

bernatnan commented on pyg_pymes.xml:
```diff
@@ -167,8 +167,8 @@ http://www.mjusticia.gob.es/cs/Satellite/1292342988403?blobheader=application%2F
-            <field name="current_value"></field>
-            <field name="previous_value"></field>
+            <field name="current_value">balance(760,761,762,767,769)</field>
+            <field name="previous_value">balance(760,761,762,767,769)</field>
```
Revisar que no estigui dns de "a) Imputación de subvenciones, donaciones y legados de carácter financiero"

juanjo-nan replied:
NO está dins.

juanjo-nan committed:
Add new financial statements to with the new update (2021), undo the changes from the old financial statements (2008).

Task #065432

bernatnan reviewed (approved):
LGT

juanjo-nan committed:
Changes in balance_pymes_2021.
In "III. Reservas" delete account 115, because this account dont exist in the account templates for PYMES.

Task #065432
